kerl

```diff
@@ -2215,6 +2230,34 @@ case "$1" in
                 ;;
         esac
         ;;
+    upgrade)
+        if [ $# -ne 1 ]; then
+            upgrade_usage
+            exit 1
+        fi
+        current_kerl_path="$(which kerl)"
+        which_status=$?
+        if [ $which_status != 0 ]; then
+            install_folder="/usr/local/bin"
```

Comment:
This should probably be its own environment variable or otherwise settable to a directory from the command line. (For example, I install _my_ kerl at `$HOME/bin/kerl`)

Reply:
Current kerl is $0 ?

Reply:
@jadeallenx, agreed. If not set, would the current choice (`/usr/local/bin`) be acceptable as a default? Should we document this?

Reply:
> Current kerl is $0 ?

I didn't understand this, sorry. `$0` is "current script".

Here (in this place in the code), we're covering the option where `kerl` is not yet "installed" anywhere (eg. local Git repo), so @jadeallenx's suggestion makes sense, but maybe I missed something (?)